libs/platform/src/lib/components/select/select.component.ts

```diff
@@ -11,7 +11,7 @@ import {
     ElementRef
 } from '@angular/core';
 import { NG_VALUE_ACCESSOR } from '@angular/forms';
-import { SelectComponent as fdSelect, SelectProxy } from '@fundamental-ngx/core';
+import { DynamicComponentService, SelectComponent as fdSelect, SelectProxy } from '@fundamental-ngx/core';
```

Comment:
I just assume this is only for compatibility reason so it does compile as we inherit select. we need to change this part..